src/analysisd/rules.c

```diff
@@ -137,6 +137,9 @@ int Rules_OP_ReadRules(const char *rulefile)
 
     const char *xml_options = "options";
 
+    const char *xml_mitre = "mitre";
```

Comment:
Changes made at this function have to be replicated on `OS_ReadXMLRules()`:
https://github.com/wazuh/wazuh/blob/a439783ace9c828238e065dfc01b90a1e31cc771/src/shared/rules_op.c#L28

For the DB output.